test/e2e/testdata/oidc-securitypolicy-backendcluster.yaml

```diff
@@ -65,9 +66,7 @@ spec:
               maxInterval: 5s
           retryOn:
             triggers: ["5xx", "gateway-error", "reset"]
-      issuer: "http://keycloak.gateway-conformance-infra/realms/master"
-      authorizationEndpoint: "http://keycloak.gateway-conformance-infra/realms/master/protocol/openid-connect/auth"
-      tokenEndpoint: "http://keycloak.gateway-conformance-infra/realms/master/protocol/openid-connect/token"
+      issuer: "https://keycloak.gateway-conformance-infra/realms/master"   # Test fetching auth endpoint from the issuer url
```

Comment:
The tokenEndpoint and authorizationEndpoint are now fetched by the EG. This is used to verify that EG uses the CA from BackendTLSPolicy when connecting to the OIDC provider's well-known endpoint.